### DISCUSSION TITLE
Problem faced while working on ERP Generic Connector

### DISCUSSION THREAD

**Discussion by Drayang** [frePPLe/frepple#483] [Q&A]
I had downloaded the Frepple via two ways which are

- Docker image ( https://github.com/frePPLe/frepple/pkgs/container/frepple-community )
- debian package (followed the guide here https://frepple.org/docs/current/installation-guide/linux-binaries.php)

There are some doubt on myside and I appreciate if you could answer the question.

Docker image( in windows OS):
 In order to enable the ERP connector, I need to uncomment the  "freppledb.erpconnection" in djangosettings.py. However, I do not have the source code, i.e the full open source code, since i was just pulling the image from the github and installed it using docker desktop. In this way, how can i enable the ERP connector? After that how can i edit the erp2frepple and frepple2erp source code?

**Comment by jdetaeye** [comment]
In an earlier answer, I already mentioned that inheriting from the standard frepple image is the correct approach to add you customizations.
The documentation even has an example dockerfile:
https://frepple.org/docs/current/installation-guide/docker-container.php#deployment-with-custom-extension-apps

**Reply by Drayang** [reply]
Thank you for the reply. My aim of doing all these installation and asking so many question is to eventually implement the ERP generic connector for the integration between ERPnext and Frepple. I am interested to integrate both system and would like to make this as a new connector for Frepple like Odoo Connector (Possible can post the idea as an issue and begin the development). However, i am still a newbie to Frepple and just start working on ERPNext for 2 months. Hence, any help for your side is appreciated!

So far the only method that i can manage to edit the djangosettings.py file and enable the connector is by using the debian package installation, because it is the only way I can have the full source code file in my PC. I assume i have did it correctly as I can saw the "import data from erp" option in the execute page. Following the guide here:
https://frepple.org/docs/current/integration-guide/generic-erp-connector.php 
To use the connector, the next step is to edit both frepple2erp and erp2frepple file. However, i am wondering how can frepple connect to my erp system database because the documentation do not mention about this. Did i miss out any documentation regarding the configuration for the connection set up? Can you provide some simple guide or flow on how to implement the erp connector? 

Thank you a lot for your help again! :)
